tests/override_config_table/test_override_config_table_masic.py

```diff
@@ -0,0 +1,250 @@
+        backup_config(duthost, config, config_backup)
+    backup_config(duthost, CONFIG_DB, CONFIG_DB_BACKUP)
+    # Backup Golden Config if exists.
+    for asic_id in duthost.get_frontend_asic_ids():
+        golden_config = "/etc/sonic/golden_config_db{}.json".format(asic_id)
+        golden_config_backup = "/etc/sonic/golden_config_db{}.json_before_override".format(asic_id)
+        if file_exists_on_dut(duthost, golden_config):
+            backup_config(duthost, golden_config, golden_config_backup)
+    if file_exists_on_dut(duthost, GOLDEN_CONFIG):
+        backup_config(duthost, GOLDEN_CONFIG, GOLDEN_CONFIG_BACKUP)
+
+    # Reload test env with minigraph
+    config_reload(duthost, config_source="minigraph", safe_reload=True)
+    running_config = get_running_config(duthost)
+
+    yield running_config
+
+    if topo_type in ["m0", "mx"]:
+        update_pfcwd_default_state(duthost, "/etc/sonic/init_cfg.json", original_pfcwd_value)
+    # Restore configDB after test.
+    for asic_id in duthost.get_frontend_asic_ids():
+        config = "/etc/sonic/config_db{}.json".format(asic_id)
+        config_backup = "/etc/sonic/config_db{}.json_before_override".format(asic_id)
+        restore_config(duthost, config, config_backup)
+    restore_config(duthost, CONFIG_DB, CONFIG_DB_BACKUP)
+    # Restore Golden Config after test, else cleanup test file.
+    for asic_id in duthost.get_frontend_asic_ids():
+        golden_file = "/etc/sonic/golden_config_db{}.json".format(asic_id)
+        golde_file_backup = "/etc/sonic/golden_config_db{}.json_before_override".format(asic_id)
+        if file_exists_on_dut(duthost, golde_file_backup):
+            restore_config(duthost, golden_file, golde_file_backup)
+        else:
+            duthost.file(path=golde_file_backup, state='absent')
+    if file_exists_on_dut(duthost, GOLDEN_CONFIG_BACKUP):
+        restore_config(duthost, GOLDEN_CONFIG, GOLDEN_CONFIG_BACKUP)
+    else:
+        duthost.file(path=GOLDEN_CONFIG, state='absent')
+    # Restore config before test
+    config_reload(duthost)
+
+
+def load_minigraph_with_golden_empty_input(duthost):
+    """Test Golden Config with empty input
+    """
+    initial_host_config = get_running_config(duthost)
+    initial_asic0_config = get_running_config(duthost, "asic0")
+
+    empty_input = {}
+    reload_minigraph_with_golden_config(duthost, empty_input)
+
+    # Test host running config override
+    host_current_config = get_running_config(duthost)
+    for table in initial_host_config:
+        if table in NON_USER_CONFIG_TABLES:
+            continue
+        pytest_assert(
+            initial_host_config[table] == host_current_config[table],
+            "empty input compare fail! {}".format(table)
+        )
+
+    # Test asic0 running config override
+    asic0_current_config = get_running_config(duthost, "asic0")
+    for table in initial_asic0_config:
+        if table in NON_ASIC_CONFIG_TABLES:
```

Comment:
The table in NON_ASIC do exists in asic's config due to historic minigraph parsing, though those tables should not exits.
As empty golden config means no change in configdb. Ignore `NON_USER_CONFIG_TABLES` should be enough.

Reply:
just validated back, it's ok to skip only NON_USER as well, probably previous config was messed up. 
for empty override nothing should be changed so keys should remain the same before&after